### DISCUSSION TITLE
"about" page in admin panel

### DISCUSSION THREAD

**Discussion by Joduai** [danpros/htmly#822] [General]
Most CMS admin panels contain an "about" page with basic info, some credits about author(s), along with useful links.  
Wouldn't it be useful to add such page? Either as an additional page in menu, or as a tab in config.  
Page could contain information like
- current version (moved from the bottom of admin theme);
- link to htmly website or straight to download
- link to author (website or gh); maybe along with sponsorship button?
- link to docs, and discussion forum (on gh)
- maybe info about who made translation of currently chosen language 
- backup reminder if last backup was created X time ago (no idea, 3 or 6months);
- things like website "health" (security?) , showing how much space website takes up, some server info like current php version, maybe some alerts (when certain directories have not recommended access rights)

any thoughts about that or additional elements?

**Comment by danpros** [comment]
Thanks. This is good idea, we should add this about page the the core.
